webrtc.html

```diff
@@ -11221,8 +11222,6 @@ <h2>
                 <p>
                   {{setCodecPreferences}} will reject attempts to set <var>codecs</var>
                   [= codec match | not matching =] codecs found in
-                  {{RTCRtpSender}}.{{RTCRtpSender/getCapabilities}}(<var>kind</var>)
-                  or
```

Comment:
Shouldn't the transceiver direction filtering when generating the SDP already take care of this? https://github.com/w3c/webrtc-pc/issues/2932#issuecomment-1933628242

If we don't allow preferring sendonly codecs for the sendonly transceiver usecase, then it's impossible to do an O/A exchange that results in a sendonly codec WITHOUT having to modify SDP

Reply:
Nevermind